docs/anonymizer/index.md

```diff
@@ -5,6 +5,9 @@ entities with desired values.
 
 Persidio anonymizer comes with predefined operators but can easily be extended.
 
+Presidio-Anonymizer supports both anonymization and deanonymization of encrypted entities.
```

Comment:
Consider creating a paragraph out of these three sentences, making the reading more fluent

Reply:
Done.